requirements.txt

```diff
@@ -1,10 +1,11 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
-# 
+#
 # FIXME: Packing the charm with 2.2.0+139.gd011d92 will not include dependencies in PYDEPS key:
 # https://chat.charmhub.io/charmhub/pl/wngp665ycjnb78ar9ojrfhxjkr
 # That's why we are including cosl here until the bug in charmcraft is solved
-cosl
+#cosl
+git+https://github.com/canonical/cos-lib@feature/mandatory-relation-pairs
```

Comment:
Why do not we wait for cosl to be released to pypi so we do not have to create another PR ?

Reply:
Yep, done!